package-lock.json

```diff
@@ -14760,9 +14760,9 @@
       }
     },
     "tree-kill": {
-      "version": "1.2.1",
-      "resolved": "https://registry.npmjs.org/tree-kill/-/tree-kill-1.2.1.tgz",
-      "integrity": "sha512-4hjqbObwlh2dLyW4tcz0Ymw0ggoaVDMveUB9w8kFSQScdRLo0gxO9J7WFcUBo+W3C1TLdFIEwNOWebgZZ0RH9Q==",
+      "version": "1.2.2",
```

Comment:
I'm fine with that update, but that's strange, that without change in `package.json` it ugraded one package.

Reply:
Probably just `npm install`. Without additional flag it updates package-lock automatically

Reply:
but if `package-lock.json` is present and there's no change in `package.json` (which is a case here), there shouldn't be updates in `package-lock.json` (beyond widely known problem with overwriting `http` with `https` and vice versa).

But that's still a theoretical problem, as I'm fine with this update here.